### PR TITLE
Update markerclusterer - moved to GitHub from SVN.

### PR DESCRIPTION
**Pull request by oana-sipos**
There was an issue with the markers on the Events map. I think it has to do with the fact that Google moved the repo from SVN to GitHub. I have updated the file + added the 5 markers (m1-m5). The images were added in folder `static/img/*` - I find it weird there was no other markers before, so if you spot a mistake, please flag it and don't accept the PR. Thanks!

### CODE REVIEW

**Comment by mitio** [comment]
Hey @oana-sipos,

It's great that you spotted the issue and provided a fix!

You're probably right that the images stopped working since this URL is no longer available: `http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/images/m2.png` I think we shouldn't have used that in the first place.

After a quick glance over the diff I think it looks fine and your solution to include the images in our repo is a lot better than to reference them from an external server.

I can't do much more, however, since I've stepped down from the webdev team at Code Week. I will give you write access to the repositories and you'll be able to merge this PR yourself should you decide so. I can also provide you with access to the production server so you or someone esle can deploy the fix.

File: static/js/markerclusterer.js [hunk 1]
```diff
@@ -191,9 +190,7 @@ function MarkerClusterer(map, opt_markers, opt_options) {
  * @type {string}
  * @private
  */
-MarkerClusterer.prototype.MARKER_CLUSTER_IMAGE_PATH_ =
-    'http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/' +
-    'images/m';
+MarkerClusterer.prototype.MARKER_CLUSTER_IMAGE_PATH_ = '../images/m';
```
Comment (mitio) [inline static/js/markerclusterer.js]: Just one more thing on this change:

If `markerclusterer.js` is a third-party library and if you've updated the file with the latest version of the library, we shouldn't modify it in place. If the change on this line, regarding the path to the images set in `MARKER_CLUSTER_IMAGE_PATH_` is made by you, you should revert it and leave it to the default and then set the proper value outside of this file, in another JS file.

**Comment by kerstin** [comment]
@mitio Just saw this and invited @oana-sipos to be a member of the org.

**Comment by mitio** [comment]
@kerstin Yeah, I also added her in the "Members" team (people with write and admin access).

If you're interested into restructuring the teams, please go ahead as you see fit. 🚢

File: static/js/markerclusterer.js (hunk 1 above)
Comment (oana-sipos) [inline static/js/markerclusterer.js]: You have a point. Will take a closer look at it tonight and revert it. Thanks!

**Comment by oana-sipos** [comment]
@mitio Could you please deploy? Thanks 🍨

**Comment by mitio** [comment]
@oana-sipos I deployed it using the `./deploy-latest-coding-events` script I wrote (as mentioned in the [instructions](https://gist.github.com/mitio/13e3d2ded240e434cbd5)).

Now no markers show on the map, however.

I will leave it deployed so you can track what might have caused it. I did a very quick investigation but I see no apparent problems aside from the fact that the `markersclusterer.js` is loaded from this URL in production: `http://events.codeweek.eu/static/CACHE/js/4ed9097d9207.js` which means the images would be searched for in `http://events.codeweek.eu/static/CACHE/img/m...` which is not correct. Maybe an absolute path needs to be used for them, e.g. `/static/img/m`. I think, however, that the problem is somewhere else since no marker images are attempted to be loaded and no markers appear to be drawn at all.

**Comment by mitio** [comment]
P.S. Here is [the deploy log](https://gist.github.com/mitio/90b78dbb4475c563b2f128e7b2620e3b) and the short [deployment script](https://gist.github.com/mitio/ebd22257cfb9a53d115471835d56750d).

**Comment by oana-sipos** [comment]
Something is clearly weird in here. As far as I understand `markerclusterer.js` is loaded in production from `http://events.codeweek.eu/static/CACHE/js/8121d734f1ac.js` which doesn't fully make sense. I am trying to run everything locally and also deploy to heroku to see how it works in a prod env. An evening + night of traveling is awaiting for me, so it might take until tomorrow to fix it fully (am afraid will not have a connection anytime soon). If you think it's better, please revert the deploy to the previous commit. Thanks, keep in touch!
p.s. About deploying: guess you would need a private ssh key from me to be able to connect to root?

**Comment by mitio** [comment]
Things are different on the public site due to the fact that there's some concatenation and caching going on only on production. You might need to check out the actual configuration on the production server.

However, the file itself is being loaded, so that doesn't seem to be the problem. The issue is possibly somewhere else.

Let's leave this version up for now. No need to revert back to another broken version.

For deploying, I will need your **public** key, not the private one. The private key must stay only in your possession. You can check out [this article](https://help.ubuntu.com/community/SSH/OpenSSH/Keys) or [this one](https://www.digitalocean.com/community/tutorials/how-to-set-up-ssh-keys--2) on the topic on generating the private-public key pair. When you've generated one, just email me the contents of your public key (most likely contained in `~/.ssh/id_rsa.pub`).

**Comment by oana-sipos** [comment]
I meant public of course :P E-mail sent to you with it. Just ping me when you add it. Thanks!

**Pull request merged**